### PR TITLE
add getServiceInfo method

### PR DESCRIPTION
**Pull request by ilya-ugr**

### CODE REVIEW

File: tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java [hunk 1]
```diff
@@ -46,4 +51,24 @@ private Long parseRunId(String id) {
             throw new IllegalArgumentException(e);
         }
     }
+
+    @Override
+    public TesServiceInfo getServiceInfo() {
+        String nameOfService = "CloudPipeline";
```
Comment (mzueva) [inline tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java]: move these values to application properties and inject them into service using `@Value` annotation

File: tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java [hunk 2]
```diff
@@ -46,4 +51,24 @@ private Long parseRunId(String id) {
...
+        String doc = "https://epam.github.io/cloud-pipeline/";
+        TesServiceInfo tesServiceInfo = new TesServiceInfo();
+        tesServiceInfo.setName(nameOfService);
+        tesServiceInfo.setDoc(doc);
+        tesServiceInfo.setStorage(getDataStorage());
+        return tesServiceInfo;
+    }
+
+    private List<String> getDataStorage(){
+        List<String> listPathDataStorage = new ArrayList<>();
```
Comment (mzueva) [inline tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java]: Let's use Stream API here:
`cloudPipelineAPIClient.loadAllDataStorages().stream().map( ... ).collect( ... )`
Also check that `cloudPipelineAPIClient.loadAllDataStorages()` doesn't return `null`. Convenient way to use it `ListUtils.emptyIfNull` method

**Review by mzueva — CHANGES_REQUESTED** [review]

File: tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java [hunk 3]
```diff
@@ -46,4 +62,19 @@ private Long parseRunId(String id) {
             throw new IllegalArgumentException(e);
         }
     }
+
+    @Override
+    public TesServiceInfo getServiceInfo() {
+        Stream<TesServiceInfo> tesServiceInfoStream = Stream.of(new TesServiceInfo())
```
Comment (SilinPavel) [inline tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java]: I think that it is an over-engineering here, why you suppose that using streams here is a good way? Lets Create this object just with old fashion java:
````
TesServiceInfo tesServiceInfo = new TesServiceInfo();
tesServiceInfo.set....
tesServiceInfo.set....
tesServiceInfo.set....
return tesServiceInfo
````

**Review by SilinPavel — CHANGES_REQUESTED** [review]

File: tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java [hunk 4]
```diff
@@ -1,17 +1,32 @@
...
+import org.springframework.context.annotation.PropertySource;
 import org.springframework.stereotype.Service;
 import org.springframework.util.Assert;
 
+import java.util.List;
+import java.util.stream.Collectors;
+
 @Slf4j
 @Service
+@PropertySource("application.properties")
```
Comment (SilinPavel) [inline tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java]: do we really need here this line? I believe that Spring will search this file for properties by default?

File: tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java [hunk 5]
```diff
@@ -1,17 +1,30 @@
...
 import org.springframework.stereotype.Service;
 import org.springframework.util.Assert;
 
+import java.util.List;
+import java.util.stream.Collectors;
+
 @Slf4j
 @Service
 public class TesTaskServiceImpl implements TesTaskService {
+    @Value("cloud.pipeline.service.name")
```
Comment (SilinPavel) [inline tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java]: you forget here ${}

File: tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java [hunk 6]
```diff
@@ -1,17 +1,30 @@
...
+    private String nameOfService;
+
+    @Value("cloud.pipeline.doc")
```
Comment (SilinPavel) [inline tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java]: same

File: tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java [hunk 7]
```diff
@@ -46,4 +59,18 @@ private Long parseRunId(String id) {
             throw new IllegalArgumentException(e);
         }
     }
+
+    @Override
+    public TesServiceInfo getServiceInfo() {
+        TesServiceInfo tesServiceInfo = new TesServiceInfo();
```
Comment (SilinPavel) [inline tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java]: final

File: tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java [hunk 8]
```diff
@@ -46,4 +59,18 @@ private Long parseRunId(String id) {
...
+        tesServiceInfo.setName(nameOfService);
+        tesServiceInfo.setDoc(doc);
+        tesServiceInfo.setStorage(getDataStorage());
+        return tesServiceInfo;
+    }
+
+    private List<String> getDataStorage() {
+        List<AbstractDataStorage> storageList = cloudPipelineAPIClient.loadAllDataStorages();
```
Comment (SilinPavel) [inline tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java]: you don't need local variable here just do:
```
return ListUtils.emptyIfNull(cloudPipelineAPIClient.loadAllDataStorages())
              .stream()
              .map(storage -> storage.getPath())
              .collect(Collectors.toList());
```

**Review by SilinPavel — CHANGES_REQUESTED** [review]

**Review by SilinPavel — APPROVED** [review]

**Pull request merged**